packaging/docker/entrypoint.py

```diff
@@ -71,13 +71,15 @@ def load_config_from_env() -> Dict[str, Any]:
     sleep_secs = os.environ.get('SLEEP_SECS')
     max_size_in_mb_all_logs = os.environ.get('MAX_SIZE_IN_MB_ALL_LOGS')
     max_logfiles_num = os.environ.get('MAX_LOGFILES_NUM')
+    sqlite_instructions = os.environ.get('SQLITE_VM_INSTRUCTIONS')
```

Comment:
Why not name it the env var the same?

Reply:
I'll just stick to `SQLITE_INSTRUCTIONS` then.

Reply:
Yes it would be better to keep the same name